src/js/toolbar.js

```diff
@@ -356,6 +356,7 @@ var Toolbar;
         showAndUpdateToolbar: function () {
             this.modifySelection();
             this.setToolbarButtonStates();
+            this.base.trigger('updateToolbar', {}, this.base.getFocusedElement());
```

Comment:
Ok, if that's the case then I would likely rename this event and ideally add some sort of test to make sure these kind of restrictions aren't broken in the future.  Maybe `toolbarButtonStateUpdate` or something like that? This event seems much more targeted than just a toolbarUpdate no?